js/main.js

```diff
@@ -92,4 +116,32 @@ $entriesNavAnchor.addEventListener('click', function (event) {
   $entryFormEntireDiv.className = 'entry-form-entire-div hidden';
 
   $entriesEntireDiv.className = 'entries-entire-div';
+
+  $entryForm.reset();
+  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+  editTime = false;
+});
+
+$entry.addEventListener('click', function (event) {
+
+  if (event.target.tagName === 'I') {
+    var $closestEntry = event.target.closest('.container');
+
+    document.querySelector('.new-entry').textContent = 'Edit Entry';
+    editTime = true;
+
+    $entryFormEntireDiv.className = 'entry-form-entire-div';
+    $entriesEntireDiv.className = '.entries-entire-div hidden';
+
+    var closestEntryIDNum = parseInt($closestEntry.getAttribute('data-entry-id'));
+    var closestEntryEntriesIndex = data.entries.length - closestEntryIDNum;
```

Comment:
You should not depend on an `id` to be the same as its index like this. For instance, what if eventually the item with the id `4` gets deleted? Now this won't work. Instead, you should loop over your list of entries and find the entry with the matching id.